app/controllers/publishers_controller.rb

```diff
@@ -321,9 +321,8 @@ def show
   # Domain verified. See balance and submit payment info.
   def home
     if current_publisher.promo_stats_status == :update
-      PublisherPromoStatsFetcher.new(publisher: current_publisher).perform
+      SyncPublisherPromoStatsJob.perform_later(publisher: current_publisher)
```

Comment:
I don't think this matches with the parameters of `perform`

Reply:
Hm, not sure I follow.  `SyncPublisherPromoStatsJob` takes two optional parameters, `active_promo_id` and `publisher`.  Unless specified, `active_promo_id` will be the default set by the config var, and if `publisher` is not supplied, the job will run for all publishers.